### PR TITLE
python3Packages.sentry-sdk: disable tests

### LINKED ISSUE #169130: `python39Packages.sentry-sdk` build failure on x86_64-linux as of `891d0226`
## Issue description
Build of `python39Packages.sentry-sdk` failed on x86_64-linux as of 891d0226de6e281422a8d0d99814925fd7aed8d0. This is currently breaking python3Packages.wandb.

```
error: builder for '/nix/store/j4lfky79xqjiabprrqzba2d8mjm6rhf4-python3.9-sentry-sdk-1.5.8.drv' failed with exit code 1;
       last 10 log lines:
       > ERROR tests/integrations/flask/test_flask.py::test_errors_not_reported_twice[integrations0]
       > ERROR tests/integrations/flask/test_flask.py::test_errors_not_reported_twice[integrations1]
       > ERROR tests/integrations/flask/test_flask.py::test_logging - werkzeug.excepti...
       > ERROR tests/integrations/flask/test_flask.py::test_500 - werkzeug.exceptions....
       > ERROR tests/integrations/flask/test_flask.py::test_error_in_errorhandler - we...
       > ERROR tests/integrations/flask/test_flask.py::test_tracing_success - werkzeug...
       > ERROR tests/integrations/flask/test_flask.py::test_tracing_error - werkzeug.e...
       > ERROR tests/integrations/flask/test_flask.py::test_class_based_views - werkze...
       > ERROR tests/integrations/flask/test_flask.py::test_sentry_trace_context - wer...
       > = 37 failed, 563 passed, 32 skipped, 18 deselected, 2 xfailed, 17 warnings, 38 errors in 75.70s (0:01:15) =
       For full logs, run 'nix log /nix/store/j4lfky79xqjiabprrqzba2d8mjm6rhf4-python3.9-sentry-sdk-1.5.8.drv'.
error: 1 dependencies of derivation '/nix/store/ygwhmibr8flr7xxxnif44w2wbi8i9fvq-python3.9-wandb-0.12.14.drv' failed to build
```

[full build log](https://github.com/samuela/nixpkgs-upkeep/actions/runs/2181534947)

python3Packages.wandb maintainers cc: @samuela
python39Packages.sentry-sdk maintainers cc: @fabaff @gebner
Other cc: n/a

### Steps to reproduce
1. Checkout nixpkgs at commit 891d0226de6e281422a8d0d99814925fd7aed8d0
2. Run `nix-build -A python39Packages.sentry-sdk`

## Technical details
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.13.0-1021-azure, Ubuntu, 20.04.4 LTS (Focal Fossa), nobuild`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.7.0`
 - nixpkgs: `/nix/store/r902mrj2iwsjhy8gs6y0rbr9nada0z4z-source`
```

## Misc.
This issue was automatically generated by [nixpkgs-upkeep](https://github.com/samuela/nixpkgs-upkeep).
- [CI workflow](https://github.com/samuela/nixpkgs-upkeep/actions/runs/2181534947) that created this issue.
- Internal tag: 3211800932645290548

### PR DESCRIPTION
**Pull request by gador**
###### Description of changes

disable tests in the wake of Flask and Werkzeug update. Also disable failing tests on non-x86_64-linux architecture.

Fixes #169130

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

This is a semi-automatic executed nixpkgs-review with nixpkgs-review-checks extension. It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of nixpkgs-review run on x86_64-linux 1

<details>
<summary>
3 packages marked as broken and skipped:
</summary>
<ul>
<li>python39Packages.spacy</li>
<li>python39Packages.spacy-transformers</li>
<li>python39Packages.textacy</li>
</ul>
</details>
<details>
<summary>3 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>alerta-server: log was empty</li>
<li>gns3-gui: log was empty</li>
<li>gns3-server: log was empty</li>
  </ul>
</details>
<details>
<summary>
11 packages built:
</summary>
<ul>
<li>cura</li>
<li>home-assistant-component-tests.sentry</li>
<li>moodle-dl</li>
<li>moz-phab</li>
<li>netbox</li>
<li>openai (python39Packages.openai)</li>
<li>python39Packages.django-rq</li>
<li>python39Packages.sentry-sdk</li>
<li>python39Packages.spacy-loggers</li>
<li>python39Packages.wandb</li>
<li>tribler</li>
</ul>
</details>

### CODE REVIEW

**Comment by gador** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 169414` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>6 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>alerta-server: log was empty</li>
<li>gns3-gui: log was empty</li>
<li>gns3-server: log was empty</li>
<li>openai: log was empty</li>
<li>python39Packages.spacy-loggers: log was empty</li>
<li>python39Packages.wandb: log was empty</li>
  </ul>
</details>
<details>

  <summary>2 packages failed to build and are new build failures:</summary>
  <ul>
<li>netbox: log was empty</li>
<li>(python39Packages.openai): log was empty</li>
  </ul>
</details>
<details>
<summary>
7 packages built:
</summary>
<ul>
<li>cura</li>
<li>home-assistant-component-tests.sentry</li>
<li>moodle-dl</li>
<li>moz-phab</li>
<li>python39Packages.django-rq</li>
<li>python39Packages.sentry-sdk</li>
<li>tribler</li>
</ul>
</details>

File: pkgs/development/python-modules/sentry-sdk/default.nix [hunk 1]
```diff
@@ -128,7 +130,17 @@ buildPythonPackage rec {
...
+    "tests/integrations/flask/test_flask.py"
+    "tests/integrations/bottle/test_bottle.py"
+    "tests/integrations/django/test_basic.py"
+    "tests/integrations/pyramid/test_pyramid.py"
+  ]
+  # test crashes on aarch64
+  ++ (if stdenv.buildPlatform != "x86_64-linux" then [
+    "tests/test_transport.py"
+    "tests/integrations/threading/test_threading.py"
+  ] else [ ]);
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/sentry-sdk/default.nix:143]:
```suggestion
  ++ lib.optionals (stdenv.buildPlatform != "x86_64-linux") [
    "tests/test_transport.py"
    "tests/integrations/threading/test_threading.py"
  ];
```

**Review by SuperSandro2000 — DISMISSED** [review]

**Comment by gador** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 169414` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>6 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>alerta-server: log was empty</li>
<li>gns3-gui: log was empty</li>
<li>gns3-server: log was empty</li>
<li>openai: log was empty</li>
<li>python39Packages.spacy-loggers: log was empty</li>
<li>python39Packages.wandb: log was empty</li>
  </ul>
</details>
<details>

  <summary>2 packages failed to build and are new build failures:</summary>
  <ul>
<li>netbox: log was empty</li>
<li>(python39Packages.openai): log was empty</li>
  </ul>
</details>
<details>
<summary>
7 packages built:
</summary>
<ul>
<li>cura</li>
<li>home-assistant-component-tests.sentry</li>
<li>moodle-dl</li>
<li>moz-phab</li>
<li>python39Packages.django-rq</li>
<li>python39Packages.sentry-sdk</li>
<li>tribler</li>
</ul>
</details>

**Review by fabaff — APPROVED** [review]
Looks good to me.

**Pull request merged**